src/wdio/conf.js

```diff
@@ -0,0 +1,37 @@
+const AxeService = require('./services').Axe;
+const TerraService = require('./services').Terra;
+const visualRegression = require('./visualcompare');
+const SeleniumDockerService = require('./services').SeleniumDocker;
+const path = require('path');
+
+exports.config = {
+  specs: [path.join('.', 'tests', 'specs', '**', '*.js')],
+
+  maxInstances: 10,
+
+  capabilities: [
+    {
+      browserName: 'chrome',
```

Comment:
The Selenium Service that handles which browser is used (chrome/firefox), should this be applied in the axe service such that the testing will be consistent by default?

Reply:
> should this be applied in the axe service such that the testing will be consistent by default?

What do you mean?

Reply:
Okay. Missed these config overlaps.

What I was thinking would have been axe and selenium should use the same config for testing in either chrome or firefox. Missed the fact the wdio config drives the selenium config for this.